app/models/gobierto_data/dataset.rb

```diff
@@ -43,6 +57,14 @@ def available_formats
       [:csv]
     end
 
+    def dataset_path
+      url_helpers.gobierto_data_root_path.concat('/').concat(slug)
```

Comment:
Why not using the dataset route path named `gobierto_data_datasets` ??

If you use the root _path and manually modify is the same as not using any route

Reply:
when I passed /data/4 was redirected because should use the slug. But thanks to you I learned that I can pass a parameter to replace :id like this.
```
gobierto_data_datasets_path(id: self.slug)
```
thx